lib/util.js

```diff
@@ -227,8 +226,7 @@ module.exports = {
     this.regexCache[name] = this.regexCache[name] || new RegExp('(?:\\/\\*\\s*(?:!)?\\s*rtl:' + (name ? escapeRegExp(name) + '(?::)?' : '') + ')([^]*?)(?:\\*\\/)', 'img')
     return this.regexCache[name]
   },
-  regex (what, options) {
-    what = what || []
+  regex (what = [], options) {
```

Comment:
The first `regex` parameter is always true AFAICT, so maybe we should remove the fallback while at it?